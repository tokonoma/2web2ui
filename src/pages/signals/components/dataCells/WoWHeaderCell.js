import React from 'react';
import { InfoOutline } from '@sparkpost/matchbox-icons';
import { Tooltip } from 'src/components/matchbox';
import useUniqueId from 'src/hooks/useUniqueId';
import useHibanaOverride from 'src/hooks/useHibanaOverride';
import OGStyles from './DataCell.module.scss';
import hibanaStyles from './DataCellHibana.module.scss';

const WoWHeaderCell = () => {
  const styles = useHibanaOverride(OGStyles, hibanaStyles);
  const id = useUniqueId('wow-header-cell');

  return (
    <Tooltip id={id} content="Week Over Week Change" dark>
      WoW <InfoOutline size={15} className={styles.InfoIcon} />
    </Tooltip>
  );
};

export default WoWHeaderCell;
