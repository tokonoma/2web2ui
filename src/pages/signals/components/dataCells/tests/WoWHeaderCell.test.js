import React from 'react';
import { shallow } from 'enzyme';
import WoWHeaderCell from '../WoWHeaderCell';

jest.mock('src/hooks/useHibanaOverride', () => styles => styles);

describe('WoWHeaderCell', () => {
  it('renders', () => {
    expect(shallow(<WoWHeaderCell />)).toMatchSnapshot();
  });
});
