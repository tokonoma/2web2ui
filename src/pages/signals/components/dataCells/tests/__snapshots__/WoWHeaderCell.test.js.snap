// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WoWHeaderCell renders 1`] = `
<Tooltip
  content="Week Over Week Change"
  dark={true}
  id="wow-header-cell-1"
>
  WoW 
  <InfoOutline
    className="InfoIcon"
    size={15}
  />
</Tooltip>
`;
